addons/website/views/snippets/snippets.xml

```diff
@@ -666,7 +669,7 @@
     <div data-js="layout_column"
         data-selector="section, section.s_carousel_wrapper .carousel-item"
         data-target="> *:has(> .row), > .s_allow_columns"
-        data-exclude=".s_masonry_block, .s_features_grid, .s_media_list, .s_table_of_content, .s_process_steps, .s_image_gallery, .s_timeline">
+        data-exclude=".s_masonry_block, .s_features_grid, .s_media_list, .s_table_of_content, .s_process_steps, .s_image_gallery, .s_timeline, .s_card_offset">
```

Comment:
I agree that this snippet doesn't work with Column Layout 👍 
But when I tested this snippet, I realized that by doing so, we're also preventing the addition of a grid element (text/button/images).

![Capture d’écran 2024-08-23 à 10 50 27](https://github.com/user-attachments/assets/e4f9d91f-33ae-4c7d-ab3f-0f90bed36cc1)

I assume  we could always duplicate the current elements, but as there's only an image and a block of text, it would make sense to keep this option. The user might want to add a button on the image, for example.

So here's my point of view:
- (Simple) Either we decide not to prevent the layout option, so the result isn't good by switching to Column layout, but we allow the user to add grid elements in a simple way.
- (Complex) Or we find a way to make the “Add elements” option independent of the Layout option, but I'm not sure how.

What do you think about that @stefanorigano ?

Reply:
Selectors are not correctly set.

You should keep `.s_card_offset` in this `data-exlude`, but also add it the ` <!-- Grid only -->` one (a few lines up ☝️). Add it as `data-selector` , otherwise grid specific options will never be triggered ;)

```xml
<!-- Grid only -->
<div data-js="layout_column"
    data-selector="section.s_masonry_block, .s_card_offset"
    data-target="> *:has(> .row)">
    <t t-call="website.grid_layout_options"/>
</div>
```

Reply:
Nice! thanks @stefanorigano